public/assets/locales/en/translation.json

```diff
@@ -0,0 +1,168 @@
+{
+    "about_us": {
+        "meet_our_team": "Meet our team"
+    },
+    "add_item_button": {
+        "add_item": "Add Item"
+    },
+    "add_item_form": {
+        "product_info": "Product Info",
+        "note": "Your product will be added to the store after filling the required information.",
+        "item_image": "Item Image",
+        "upload_a_file": "Upload a file",
+        "or_drag_and_drop": "or drag and drop",
+        "item_name": "Item Name",
+        "item_price": "Price",
+        "item_type": "Item Type",
+        "item_category": "Item Category",
+        "item_description": "Description",
+        "adress_info": "Address Info",
+        "country": "Country",
+        "city": "City",
+        "add_new_item": "Add New Item"
+    },
+    "overlap_text": {
+        "check_out": "Watch Full Video",
```

Comment:
this one was "check out this video made by one of our satisfied customers" . i suggest we don't change it